src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,32 @@
+import { Person } from '../../types/Person';
+import './Autocomplete.scss';
+
+interface Props {
+  delay: number;
+  filteredPeople: Person[];
+  onSelected: (person: Person | null) => void;
+}
+
+export const Autocomplete: React.FC<Props> = ({
+  // delay,
```

Comment:
The `delay` prop is defined but not used. Consider using it to implement the debounce functionality for the suggestions, as per the task requirements.